cardano-config/src/Cardano/Config/Logging.hs

```diff
@@ -149,26 +125,25 @@ data LoggingFlag = LoggingEnabled | LoggingDisabled
 -- | Interpret main logging CLI controls into a tuple of:
 --   - a designation of whether logging was disabled,
 --   - a valid 'LoggingConfiguration' (still necessary, even if logging was disabled)
-loggingCLIConfiguration :: LoggingCLIArguments -> IO (LoggingFlag, LoggingConfiguration)
-loggingCLIConfiguration  lca@LoggingCLIArguments{logConfigFile = Nothing} =
+loggingCLIConfiguration :: Maybe FilePath -> Bool -> IO (LoggingFlag, LoggingConfiguration)
```

Comment:
I'm thinking of implementing a sum type for `captureMetrics` (isomorphic to bool)